### PR TITLE
Es/es query

### PR DESCRIPTION
**Pull request by esoergel**
## Technical Summary
Final processing of the query was split between two locations - this combines them, so `raw_query` and `pprint` represent the query that gets sent to ES.  Impetus was I got confused because an aggregation query displayed as having a size of `SIZE_LIMIT`, but returned no results.  This was because it was amended just before being sent.

## Feature Flag
no

### Automated test coverage

Lots of tests for this already

### QA Plan

N/A

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/es/es_query.py [hunk 1]
```diff
@@ -512,11 +506,10 @@ def doc_ids(self):
     @property
     def hits(self):
         """Return the docs from the response."""
-        raw_hits = self.raw_hits
-        if not raw_hits and self.query.uses_aggregations() and self.query._size == 0:
-            raise ESError("no hits, did you forget about no_hits_with_aggs?")
+        if self.query.uses_aggregations():
+            raise ESError("We exclude hits for aggregation queries")
```
Comment (proteusvacuum) [inline corehq/apps/es/es_query.py:510]: This reduces the snark factor of this file by 10. 👍🏽

**Review by proteusvacuum — APPROVED** [review]

File: corehq/apps/es/es_query.py (hunk 1 above)
Comment (esoergel) [reply corehq/apps/es/es_query.py:510]: Not to mention that `no_hits_with_aggs` isn't a real flag/parameter!

Comment (proteusvacuum) [reply corehq/apps/es/es_query.py:510]: I think you may have just forgotten about it.

Comment (esoergel) [reply corehq/apps/es/es_query.py:510]: :unamused:

**Review by snopoke — APPROVED** [review]

**Pull request merged**